src/Report/Formatter/TextFormatter.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace PHPSA\Report\Formatter;
+
+use PHPSA\Issue;
+
+class TextFormatter implements ReportFormatter
+{
+    /**
+     * {@inheritdoc}
+     */
+    public function reportBeginning()
+    {
+        return '';
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    public function reportEnd()
+    {
+        return '';
+    }
+
+    /**
+     * @param Issue $issue
+     *
+     * @return string The formatted issue.
+     */
+    public function formatIssue(Issue $issue)
+    {
+        if ($issue->getCheckName() === 'syntax-error') {
+            return $this->formatSyntaxError($issue);
+        }
+
+        return $this->formatNotice($issue);
+    }
+
+    private function formatSyntaxError(Issue $issue)
+    {
+        $filepath = $issue->getLocation()->getFilePath();
+        $line = $issue->getLocation()->getLineStart();
+        $codeLines = file($filepath);
+
+        $output = '<error>Syntax error:  ' . $issue->getDescription() . " in {$filepath} </error>\n\n";
+
+        $lineContent = trim($codeLines[$line-1]);
+        if (!empty($lineContent)) {
+            $output .= "<comment>\t {$lineContent} </comment>\n";
+        }
+
+        return $output . "\n";
+    }
+
+    private function formatNotice(Issue $issue)
```

Comment:
Can't we change that to:
Filename
line: issue 1
line: issue 2
next Filename
...
Also we want to add language level errors :+1: